src/V3AstNodeExpr.h

```diff
@@ -1766,6 +1728,7 @@ class AstParseRef final : public AstNodeExpr {
     string m_name;
 
 public:
+    // TODO: 'expect' is only every used for dumping and has no functional effect (sameNode unused)
     AstParseRef(FileLine* fl, VParseRefExp expect, const string& name, AstNode* lhsp = nullptr,
```

Comment:
Can we remove the `expect` parameter/member? The parser is full of it, but seems unused. (We never call `sameNode` on AstParseRef, which is the only place it's referenced)

Reply:
Please remove, was for double checking the old parser.